services.html

```diff
@@ -8,6 +8,6 @@
 <body>
     <h6>Welcome to the second design</h6>
     <a href="">home</a>
-    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Temporibus deleniti nostrum porro iure, ut voluptate quibusdam cumque architecto nemo quae vel numquam ipsam expedita eaque nihil similique quis odit. Qui.</p>
+    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Temporibus deleniti nostrum porro iure, ut voluptate  cumque architecto nemo quae vel numquam ipsam expedita eaque nihil similique quis odit. Qui.</p>
 </body>
```

Comment:
Consider using some real words, instead of Lorem. It will emphasize that your work is more important.